village/src/App.js

```diff
@@ -1,8 +1,10 @@
 import React, { Component } from 'react';
-
 import './App.css';
 import SmurfForm from './components/SmurfForm';
 import Smurfs from './components/Smurfs';
+import axios from 'axios';
```

Comment:
Antonio good use of axios imports. could you explain what the import is doing?

Reply:
grabbing the axios from the axios libraries , which we use to get and post files from other servers. In short grabbing the ajax requests